presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -2846,4 +2848,17 @@ public int getEagerPlanValidationThreadPoolSize()
     {
         return this.eagerPlanValidationThreadPoolSize;
     }
+
+    @Config("exclude-java-worker-session-properties")
+    @ConfigDescription("Exclude java-worker session properties")
```

Comment:
maybe elaborate a bit more
```suggestion
    @ConfigDescription("Exclude session properties from Java workers when running on native workers")
```